docroot/themes/humsci/humsci_basic/docs/color-pairings.md

```diff
@@ -0,0 +1,27 @@
+## Color Pairings
+A single theme can have multiple color pairings. The default pairing pairing for the Colorful Theme is Ocean, and for the Traditional Theme is Cardinal.
+
+Color palettes for each theme are set in `/humsci_basic/src/scss/settings/_variables.colorful-pairings.scss` and `/humsci_basic/src/scss/settings/_variables.traditional-pairings.scss`.
+
+A user can update the color pairing in the Drupal admin by going to Appearance / Settings / Humsci Colorful (or Humsci Traditional). Under **Theme Specific Settings** you will find **Color Pairing** options.
```

Comment:
After this sentence, can you mention how these color pairing classes get affixed the html element as a class and then set CSS variables across the site?